crates/security/src/lib.rs

```diff
@@ -90,14 +90,25 @@ pub mod pallet {
     }
 
     #[pallet::genesis_config]
-    #[derive(Default)]
-    pub struct GenesisConfig;
+    pub struct GenesisConfig {
+        initial_status: StatusCode,
+    }
+
+    #[cfg(feature = "std")]
+    impl Default for GenesisConfig {
+        fn default() -> Self {
+            Self {
+                initial_status: StatusCode::Error,
+            }
+        }
+    }
 
     #[pallet::genesis_build]
     impl<T: Config> GenesisBuild<T> for GenesisConfig {
         fn build(&self) {
+            Pallet::<T>::set_status(self.initial_status);
+
             Pallet::<T>::insert_error(ErrorCode::OracleOffline);
```

Comment:
Maybe only set this if the initial status is error?

Reply:
Should still be OracleOffline, imo. After shutdown, sudo should change to Error, and then if the oracle is online, the oracle will clear the error and set the status to online